tests/test_bgp_neighbor.rb

```diff
@@ -295,7 +295,7 @@ def test_local_as
   end
 
   def test_log_neighbor_changes
-    if node.product_id[/ios_xr|N(5|6|7)/]
+    if node.product_id[/IOSXR|N(5|6|7)/]
```

Comment:
This works for now but it'll be fragile for other XR platforms when we add those. Can we instead do:

`if platform == :ios_xr || node.product_id[N(5|6|7)]`